src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -66,6 +67,57 @@ public override IEnumerable<TimeFunction> TimeFunctions(bool requiresRoundtrippi
                 DateTimeKind.Utc);
         }
 
+        [Fact]
+        public void CopyToMillisecondPresent()
+        {
+            FileInfo input = new FileInfo(Path.Combine(TestDirectory, GetTestFileName()));
+            input.Create().Dispose();
+
+            string driveFormat = new DriveInfo(input.DirectoryName).DriveFormat;
+            if (!driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
If we're going to skip this test on HFS, it'd be better to make this a ConditionalFact that's only executed when the drive format != HFS. (And if for some reason we choose not to do that, the above file creation should be moved to within this if block... there's no benefit to creating the file if we're not going to use it.)

Reply:
will this be okay
```
private static bool isHFS = new DriveInfo(Path.GetTempPath()).DriveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);
```